src/signals/IncidentMap/components/utils/get-filtered-incidents.test.ts

```diff
@@ -1,13 +1,65 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 - 2023 Gemeente Amsterdam
 import { mockFiltersShort } from '../__test__/mock-filters'
-import { mockIncidentsShort } from '../__test__/mock-incidents'
+import { mockIncidentsWithoutIcon } from '../__test__/mock-incidents-without-icon'
 import { getFilteredIncidents } from './get-filtered-incidents'
 
+const mockFiltersAllActive = mockFiltersShort.map((filter) => {
+  if (filter.subCategories) {
+    return {
+      ...filter,
+      filterActive: true,
+      subCategories: filter.subCategories.map((subFilter) => {
```

Comment:
subcategory is actually one word and camelcase should only be used to substitute spaces, but maybe too much of a hassle.

Reply:
These are typed indeed.